docs/open-source-tizen/tools/gbs/gbs-build.md

```diff
@@ -264,160 +275,155 @@ info: Local repo can be found here:
+  - The `--exclude-from-file` option specifies a text file that contains a name list of packages to be ignored.
 
   ```bash
-  $ gbs build -A i586 tizen-packages --exclude=<pkg1>$ gbs build -A i586 tizen-packages --exclude=<pkg1>,<pkg2>$ gbs build -A i586 tizen-packages --exclude-from-file=/path/to/packages.list
+  $ gbs build -A i586 tizen-packages --exclude=<pkg1>
+  $ gbs build -A i586 tizen-packages --exclude=<pkg1>,<pkg2>
+  $ gbs build -A i586 tizen-packages --exclude-from-file=/path/to/packages.list
   ```
 
-5. Build packages based on dependencies. The --deps option enables GBS to build specific packages, together with all the related packages on which they depend.The --rdep option enables GBS to build specific packages, together with all the related packages that depend on them.
+- Build packages based on dependencies:
 
-  The specific packages can be included by the --binary-from-file option or the --binary-list option, and be excluded by the --exclude option or the --exclude-from-file option.
+  - The `--deps` option enables GBS to build specific packages, together with all related packages on which they depend.
+  - The `--rdep` option enables GBS to build specific packages, together with all related packages that depend on them.
 
-  These two options are compatible. When added at the same time, besides the specific packages, GBS will build not only the related packages on which they depend, but also all the related packages that depend on them.
+  The specific packages can be included by the `--binary-from-file` or `--binary-list` option, and excluded by the `--exclude` or `--exclude-from-file` option.
+
+  The `--deps` and `--rdep` options are compatible. When added at the same time, besides the specific packages, GBS builds not only the related packages on which they depend, but also all the related packages that depend on them.
 
   ```bash
   $ gbs build -A i586 --binary-list=<pkg1>,<pkg2> --deps
   $ gbs build -A i586 --binary-list=<pkg1>,<pkg2> --rdeps
   $ gbs build -A i586 --binary-list=<pkg1>,<pkg2> --deps --rdeps
   ```
 
-### Other useful options
+## Useful Building Options
 
-#### Install extra packages to build root
+The `gbs build` command offers some useful options:
 
---extra-packs=<pkgs list sep by comma> can be used to install extra packages:
+- Install extra packages to build a root
 
-```bash
-$ gbs build -A i586 --binary-list=<pkg1>,<pkg2> --deps
-$ gbs build -A i586 --binary-list=<pkg1>,<pkg2> --rdeps
-$ gbs build -A i586 --binary-list=<pkg1>,<pkg2> --deps --rdeps
-```
+  The `--extra-packs=<packages separated by commas>` option can be used to install extra packages:
 
-#### Keep all packages in build root
+  ```bash
+  $ gbs build -A i586 --binary-list=<pkg1>,<pkg2> --deps
+  $ gbs build -A i586 --binary-list=<pkg1>,<pkg2> --rdeps
+  $ gbs build -A i586 --binary-list=<pkg1>,<pkg2> --deps --rdeps
+  ```
 
```

Comment:
This command  seems wrong, since it does not even contain the --extra-packs

Reply:
The following is correct;
```bash
  $ gbs build -A i586 --extra-packs=<pkg1>,<pkg2> --deps
  $ gbs build -A i586 --extra-packs=<pkg1>,<pkg2> --rdeps
  $ gbs build -A i586 --extra-packs=<pkg1>,<pkg2> --deps --rdeps
```

Reply:
OK, this will be fixed